### ISSUE TITLE
license: Apache 2.0 includes a patent part

### ISSUE THREAD

**Issue by markuswagnergithub** [gintool/gin#31]
Some comparisons:
https://exygy.com/which-license-should-i-use-mit-vs-apache-vs-gpl/
https://www.quora.com/Whats-the-difference-between-Apache-v2-0-and-MIT-license-What-are-the-major-consequences-of-using-Apache-software-over-MIT
https://snyk.io/blog/mit-apache-bsd-fairest-of-them-all

Information overload: https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licenses

TLDR: yes, the MIT one is short and sweet and in plain English, but I do like the Apache 2.0 one for being just a little bit more detailed. Maybe this is something to consider.

**Comment by sandybrownlee** [comment]
Keeping MIT as it's more permissive.
